convert.go

```diff
@@ -92,8 +92,7 @@ func ByteSize(bytes uint64) string {
 
 // ToString Change arg to string
 func ToString(arg any, timeFormat ...string) string {
```

Comment:
We should also add support for slices.

Reply:
Maybe it can be done via reflect in default

Reply:
sure, but in another PR , ok?

Reply:
@efectn is it not covered by the default case? or do you mean an explicit treatment?

Reply:
> @efectn is it not covered by the default case? or do you mean an explicit treatment?

It's done by fmt.Sprint. however we can make it more performant